store/mongo/datastore_mongo.go

```diff
@@ -535,6 +535,69 @@ func (db *DataStoreMongo) UpdateDevicesGroup(
 	}, nil
 }
 
+func (db *DataStoreMongo) GetFiltersAttributes(ctx context.Context) ([]model.FilterAttribute, error) {
+	database := db.client.Database(mstore.DbFromContext(ctx, DbName))
+	collDevs := database.Collection(DbDevicesColl)
+
+	cur, err := collDevs.Aggregate(ctx, []bson.M{
+		{
+			"$project": bson.M{
+				"attributes": bson.M{
+					"$objectToArray": "$" + DbDevAttributes,
+				},
+			},
+		},
+		{
+			"$unwind": "$" + DbDevAttributes,
+		},
+		{
+			"$project": bson.M{
+				DbDevAttributesName:  "$" + DbDevAttributes + ".v." + DbDevAttributesName,
+				DbDevAttributesScope: "$" + DbDevAttributes + ".v." + DbDevAttributesScope,
+			},
+		},
+		{
+			"$group": bson.M{
+				"_id": bson.M{
+					DbDevAttributesName:  "$" + DbDevAttributesName,
+					DbDevAttributesScope: "$" + DbDevAttributesScope,
+				},
+				"count": bson.M{
+					"$sum": 1,
+				},
+			},
+		},
+		{
+			"$sort": bson.D{
+				{Key: "count", Value: -1},
+				{Key: "_id." + DbDevAttributesScope, Value: 1},
+				{Key: "_id." + DbDevAttributesName, Value: 1},
+			},
+		},
+	})
+	if err != nil {
+		return nil, err
+	}
+	defer cur.Close(ctx)
+
+	var results []bson.M
+	err = cur.All(ctx, &results)
+	if err != nil {
+		return nil, err
+	}
+
+	var attributes []model.FilterAttribute
+	for _, item := range results {
+		attributes = append(attributes, model.FilterAttribute{
+			Name:  item["_id"].(bson.M)[DbDevAttributesName].(string),
+			Scope: item["_id"].(bson.M)[DbDevAttributesScope].(string),
```

Comment:
we do have a const [DbDevId              = "_id"](https://github.com/mendersoftware/inventory/blob/master/store/mongo/datastore_mongo.go#L45)